core/services/workflow_service/models/user_project.py

```diff
@@ -0,0 +1,14 @@
+from uuid import UUID
```

Comment:
Dont use the `uuid` package here. As you are defining a `sqlalchemy` model, you should use: `from sqlalchemy.dialects.postgresql import UUID`.

Use the `uuid` package for the default value.